app/views/downloads/_download_preface.html.erb

```diff
@@ -14,7 +14,7 @@
       <% Tag::VISIBLE.each do |type| %>
         <% tags = @work.tag_groups[type] %>
         <% unless tags.blank? %>
-          <dt><%= type.constantize::NAME %>:</dt>
+          <dt><%= tags.size == 1 ? type.constantize::NAME : type.constantize::NAME.pluralize %>:</dt>
```

Comment:
```suggestion
          <dt><%= type.constantize.human_attribute_name("name_with_colon", count: tags.size) %></dt>
```
This would be more i18n friendly, it's already used by the challenge_assignment_notification mailer.

Reply:
I'm resolving this for now; as we discussed elsewhere, there are a few issues with translating this:
* In general, translating downloads will not work currently. They are cached (without language key), so everything has to be forced to `en`
* `Additional Tags` needs to be hardcoded as plural (not a big issue on its own)
* `Archive Warnings` already has a `human` name of just `Warnings`

We will need to solve for these problems eventually, but they are out of scope for now. If I missed something there, please feel free to un-resolve or ping me to do it.